src/clarity/wizard/wizard.spec.ts

```diff
@@ -466,6 +480,14 @@ describe("Wizard", () => {
             expect(tab3).toBeDefined();
             expect(tab4).toBeDefined();
         });
+
+        fit("calls the user-defined onCancel handler when the Cancel button is clicked", () => {
+            fixture.detectChanges();
+
+            spyOn(instance, "onCancel");
+            doCancel(compiled);
+            expect(instance.onCancel).toHaveBeenCalled();
```

Comment:
I think there was a misunderstanding. What I meant here is that 
```typescript
doCancel(compiled);
expect(fixture.componentInstance.hasBeenCanceled).toBe(true);
```
was absolutely fine. That's all you need to test your feature.

Here, you're testing if an _object_ (in this case an EventEmitter) has been called, which doesn't make sense.
Your initial version was good, you just needed to remove the extra test on internal implementation. 😄